Tests/Translation/Extractor/File/Fixture/MyFormType.php

```diff
@@ -43,6 +44,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('street', 'text', [
                 'label' => /** @Desc("Street") */ 'form.label.street',
                 'translation_domain' => 'address',
+                'constraints' => [
+                    new NotBlank(['message' => /** @Desc("You should fill in the street") */ 'form.street.empty_value']),
```

Comment:
Can you explain what is this `@Desc("You should fill in the street")` comment?

Reply:
Hi @goetas. `@Desc` adds a default text into the translation file, and I,ve added it to test if it works as for the other options of the form (`choices`, etc)

The feature is tested, please check the FormExtract tests.

Reply:
@goetas Is there anything else I should do on the PR?